core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -777,6 +777,10 @@ oppia.directive('conversationSkin', [
             siteAnalyticsService.registerVisitOppiaFromIframeEvent(
               $scope.explorationId);
           };
+
+          $scope.progressNavSubmit = function() {
+            $scope.$broadcast('progress-nav-submit');
```

Comment:
Use a named constant similar to EVENT_ACTIVE_CARD_CHANGED, e.g. EVENT_ANSWER_SUBMITTED_FROM_NAV. Declare the constant in the same place as EVENT_ACTIVE_CARD_CHANGED and import it where needed.

Reply:
Fixed.